test/integration/test_async.py

```
@@ -0,0 +1,52 @@
+    dashboard = await grafana.dashboard.get_dashboard(dashboard["uid"])
+    assert isinstance(dashboard, dict)
+    assert dashboard["meta"]["version"] == 1
+    assert dashboard["dashboard"]["tags"] == ["foobar"]
+
+    # Non-blocking I/O parallel fetch.
+    tasks = []
+    dashboards = await grafana.search.search_dashboards()
+    for dashboard in dashboards[:4]:
+        tasks.append(grafana.dashboard.get_dashboard(dashboard["uid"]))
+    results = await asyncio.gather(*tasks)
+    assert len(results), 1
```

Ruff (0.15.7)

[warning] Non-string literal used as assert message

(RUF040)